File: Ad-Hoc/__1000__/1030/1030.py
def steps(n , k):
    result = 0
    for i in range(2, n + 1):
        result = (result + k) % i
    return result

n = int(input())
for i in range(n):
    a, b = [int(n) for n in input().split()]
    step = steps(a, b)
    print(f"Case {i + 1}: {step + 1}")

File: Ad-Hoc/__1000__/1030/1030.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);

        int n = in.nextInt();
        in.nextLine();

        for(int i = 0; i < n; i++){
            String[] input = in.nextLine().split(" ");
            int a = Integer.parseInt(input[0]);
            int b = Integer.parseInt(input[1]);

            int step = steps(a, b);
            System.out.println("Case " + (i + 1) + ": " + (step + 1));
        }
        in.close();
    }

    public static int steps(int n, int k){
        int result = 0;
        for(int i = 2; i < n + 1; i++){
            result = (result + k) % i;
        }
        return result;
    }
}